src/main/java/hudson/remoting/Channel.java

```diff
@@ -170,8 +170,7 @@
     /**
      * Records the {@link Request}s being executed on this channel, sent by the remote peer.
      */
-    /*package*/ final Map<Integer,Request<?,?>> executingCalls =
-        Collections.synchronizedMap(new Hashtable<>());
+    /*package*/ final Map<Integer,Request<?,?>> executingCalls = new ConcurrentHashMap<>();
```

Comment:
It's odd that this one has had a threading model whereas the previous hasn't. I don't know why, but it's best to be consistent and ConcurrentHashMap makes better sense.